lefthook.yml

```
@@ -0,0 +1,66 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format sidebars
+      glob: "files/sidebars/*.yaml"
+      run: yarn content fmt-sidebars
+      stage_fixed: true
+
+    - name: Fix redirects
+      glob: "files/en-us/_redirects.txt"
+      run: yarn content fix-redirects en-US && yarn content validate-redirects en-US
+      stage_fixed: true
+
+    - name: Format non-md
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --ignore-unknown --write --cache {staged_files}
```

Curious why we replace `!*.md` with these specific types?

I guess `--ignore-unknown` is no longer necessary this way?

```suggestion
      glob: "*.{js,json,yml,yaml,css,html}"
      run: yarn prettier --write --cache {staged_files}
```

---

> Curious why we replace `!*.md` with these specific types?

I guess because we want to format those file types specifically instead of images and assets?